data/stk_config.xml

```diff
@@ -188,13 +189,7 @@
   <replay max-frames="12000" delta-t="0.100"  delta-speed="0.6"
           delta-steering="0.26" />
 
```

Comment:
remove minimap setting from stk_config -> will appear in user_config (minimap tag)